charts/posit-chronicle/README.md.gotmpl

```diff
@@ -12,9 +12,8 @@
 
```

Comment:
I think we want the `README.md` files here for readability purposes, otherwise users would have to generate the docs themselves with the [ just docs command](https://github.com/rstudio/helm/blob/main/Justfile#L26-L30). 
We mention to users in https://helm.rstudio.com/ that they should look at the `README.md` files in this repo, from the github ui they render well too if you look at https://github.com/rstudio/helm/tree/main/charts/posit-chronicle for example:
<img width="1279" alt="Screenshot 2023-12-05 at 2 47 16 PM" src="https://github.com/rstudio/helm/assets/18649003/ac353dcb-e6cd-4a30-82c0-3bc20d56eb18">

Reply:
Ah - ok. I didn't quite connect that this was a public. (Too many years working in private/proprietary repos...) Thanks for clarifying!

Reply:
and np!